index.html

```diff
@@ -193,5 +193,33 @@ <h1>About me</h1>
         </li>
     </div>
 </section>
+<div class="contact-form">
+    <h1>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+<form action="https://formspree.io/f/xbjedrnz" method="post">
+    <input type="text" id="name" name="name" maxlength="30" placeholder="Full name" required>
+    <input type="email" id="email" name="email" placeholder="Email address">
+    <textarea id="message" name="message" maxlength="500" rows="10" cols="30" placeholder="Enter text here"></textarea>
```

Comment:
- [ ] [REQUIRED] Like you added validation `required` to `name` input, also, add it to the `email` and `textarea` input elements. Currently, users can submit the form without supplying the email or text.